src/test/java/com/google/sps/NewMatchQueryServletTest.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps;
+
+import static org.mockito.Mockito.when;
+
+import com.google.sps.algorithms.MatchQuery;
+import com.google.sps.data.MatchRequest;
+import com.google.sps.servlets.NewMatchQueryServlet;
+import com.google.gson.Gson;
+import com.google.sps.data.User;
+import java.io.IOException;
```

Comment:
nit: make sure order your imports in alphab order

Reply:
fixed it, thanks for the comment!!